README.md

```diff
@@ -143,6 +149,7 @@ To get better data quality for server metrics, enable "Enhanced Monitoring" in y
 
 We currently require one Docker container per RDS instance monitored.
 
+> If you have multiple databases on the same RDS instance, you can monitor them all by specifying `DB_ALL_NAMES=1` as an environment variable
```

Comment:
I would make this not quote style (i.e. remove the leading `>`) and add a `.` at the end of the sentence to match the style.

Reply:
Updated 👍